tests/test_starfile_stack.py

```diff
@@ -18,7 +18,9 @@ def run(self, result=None):
         """Overridden run method to use context manager provided by importlib_resources"""
         with importlib_resources.path(
             tests.saved_test_data, "sample_relion_data.star"
-        ) as path:
+        ) as path, importlib_resources.path(
```

Comment:
can we parameterize this test, (or alternatively make two subclasses of a generic class) and just run two cases, once for each input file? 

I feel like writing both tests together complicates the code, and wouldn't really give distinct feedback as a unit test (which case broke it?).